<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyRubric</class>
 <widget class="QDialog" name="ModifyRubric">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of test:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_DeleteRubric">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>430</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>*Rubric Id :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_activityName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_rubricValue">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_rubricText">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_activityId">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ItemNum">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_UpdateRubric">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>430</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Activity Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_numTest">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="Rubric_table">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>531</width>
     <height>231</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>Item Number</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Item ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Item Text</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>*Rubric Item Number :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rubricID">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>370</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>*Maximum points of the iitem :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Activity Id :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>*Description of the item :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
